### ISSUE TITLE
Missing feature: Telemetry > Bluetooth > Connect to device

### ISSUE THREAD

**Issue by MrD-RC** [FrSkyRC/ETHOS-Feedback-Community#4149]
Currently, the only option is to enable Bluetooth for telemetry so that you can connect to the transmitter with a mobile phone. However this is limiting.

i would like so see the option for the transmitter to be able to connect to a device. For example a Bluetooth slave module on an antenna tracker. This would open up so much more possibilities for ETHOS. You used to be able to search for a device on the older versions of ETHOS. I remember being able to connect to a Bluetooth earbud for external audio. Please add this for telemetry.

**Comment by mawzthefinn** [comment]
BT Audio is still there, it's now at the bottom of the Audio section in System->General (if your radio has that feature)

The old Wireless menu got split into 3 places. Audio support is in System->General, Telemetry output is in Model->Telemetry and Model->Trainer gets the PARA/Trainer functionality. File transfer was added in Model Select and File Manager as well.

IIRC only PARA/Trainer supported searching for devices.

**Comment by MrD-RC** [comment]
No, the BT audio also searched for devices. Earbuds don't have a search feature. They can only be slave devices. So both Bluetooth modules can search. 

The ability to connect to other devices for telemetry is pretty important. Having the transmitter only being a slave device is limiting.

I can't see an option to enable the SmartPort connection on the top of the radio for telemetry either. If that worked. I could have fitted an external Bluetooth or WiFi module there.

**Comment by mawzthefinn** [comment]
BT Audio searches for devices on 1.5 (I just checked)

IIRC, if you turn the S.Port connector to 5V in Edit Model, it should enable S.Port mirroring as well.

**Comment by MrD-RC** [comment]
That could be handy. It would be nice as an option on the Telemetry settings too.

**Comment by OldGazer** [comment]
> That could be handy. It would be nice as an option on the Telemetry settings too.

Except for one tiny thing:  By design, BT range SUCKS.  I have to wear BT enabled hearing aids so I can hear my TV.  I walk  20 feet to the toilet and the sound goes away.  There is no way I'd ever use a BT connection to either control an aircraft or to monitor its performance.

**Comment by MrD-RC** [comment]
This is for wireless communication to things that are close. Ground stations, antenna trackers etc. So the range isn't a problem. 10m is all that's needed.

**Comment by mawzthefinn** [comment]
Just tested this a bit more on current 26.1.0

Trainer - BT Master, BT Send File, and BT audio support Search for Device, , BT Receiver File, BT Slave and Telemetry only advertise (they act as a peripheral).

I believe that's pretty much the correct set of options for BT
